components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2048,47 +2077,77 @@ private int handleStartIndexEqualsNULL(Integer startIndex) {
     }
 
     /**
-     * Get role names according to the given domain. If the domain is not specified, roles of all the user
-     * stores will be returned.
-     *
-     * @param domainName Domain name
-     * @return Roles List
+     * Get role names according to the given domain. If the domain is not specified, roles of all the user stores
+     * will be returned.
+     * @param startIndex
```

Comment:
Can we add descriptions for params, return type, and thrown exceptions in this doc comment?